kube_scheduler/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # CHANGELOG - Kube_scheduler
 
+## 1.2.0 / 2020-01-13
+
+* [Added] Make OpenMetrics use the RequestsWrapper. See [#5414](https://github.com/DataDog/integrations-core/pull/5414).
+* [Added] Move unit conversion helpers to openmetrics mixin. See [#5364](https://github.com/DataDog/integrations-core/pull/5364).
```

Comment:
```suggestion
* [Fixed] Move unit conversion helpers to openmetrics mixin. See [#5364](https://github.com/DataDog/integrations-core/pull/5364).
```

Reply:
Fixed seems more appropriate seems we are just refactoring.

Reply:
I changed the label on https://github.com/DataDog/integrations-core/pull/5364:
 `changelog/Fixed` => `changelog/Added`